test/WebJobs.Extensions.Tests/Extensions/Core/ErrorTriggerListenerTests.cs

```diff
@@ -229,11 +232,12 @@ public override string Message
                     }
                 }
 
-                public override Collection<TraceEvent> Events
+                public override IEnumerable<TraceEvent> GetEvents()
                 {
-                    get
+                    // locking to prevent race condition while accessing events
```

Comment:
nit: here and other places above, you can remove this "locking to prevent" comment. It's clear from the code :)